setup.py

```diff
@@ -72,7 +72,11 @@
             "torch",
             "transformers",
         ],
-        "local-inference": ["unstructured-inference>=0.2.4"],
+        "local-inference": [
+            "unstructured-inference>=0.2.4",
+            # NOTE(robinson) - This is temporary due to a multithreading issue
+            "unstructured-inference<0.2.8"
```

Comment:
```suggestion
            # NOTE(robinson) - Upper bound is temporary due to a multithreading issue
            "unstructured-inference>=0.2.4,<0.2.8",
```
You can merge these two, see https://stackoverflow.com/a/8811418 for an example.